common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -186,6 +186,9 @@ public class CommonValue {
             + MLModel.DEPLOY_TO_ALL_NODES_FIELD
             + "\": {\"type\": \"boolean\"},\n"
             + "      \""
+            + MLModel.IS_HIDDEN_FIELD
```

Comment:
You should also bump `ML_MODEL_INDEX_SCHEMA_VERSION` to 8. Check line 40 of this file

Reply:
ack